admin/brand.php

```diff
@@ -0,0 +1,136 @@
+<?php include("includes/header.php");
+      include('model/database.php');
+      include('model/brand.php');
+
+      
+//QUERY THE DATABASE AND STORE ALL USERS INTO A VARIABLE
+
+// Create your query
+$query = 'SELECT * FROM BRAND';
+
+// Run your query
+$result = mysqli_query($connection, $query);
+
+// Check if the database returned anything
+if($result) {
+    $brands = mysqli_fetch_all($result, MYSQLI_ASSOC);
+    print_r($brands);
+} else {
+    // Output an error
+    echo "<p>Error</p>";
+}
+
+if($_SERVER['REQUEST_METHOD'] == 'POST'){
+//add brand information to db
+if(isset($_POST['add'])) {    
+    $brand_name = $_POST['brand_name'];
+    $brand_description = $_POST['brand_description'];
+        
+    // check empty fields
+    if(empty($brand_name) || empty($brand_description)) {                
+        if(empty($brand_name)) {
+            echo "<font color='red'>Brand name field is empty.</font><br/>";
+        }
+        
+        if(empty($brand_description)) {
+            echo "<font color='red'>Brand description field is empty.</font><br/>";
+        }
+        
+        //link to the previous page
+        echo "<br/><a href='brands.php'>Go Back</a>";
+    } else { 
+        // if all the fields are filled (not empty)             
+        //insert data to database
+        $add_query = "INSERT INTO BRAND(Brand_Name, Brand_Desc) VALUES('$brand_name','$brand_description'";
+        $add_result = mysqli_query($connection, $add_query);
+        $brands = mysqli_fetch_all($add_result, MYSQLI_ASSOC);
+        print_r($result);
+        //redirect to brands list
+        // header("Location: brands.php");
+    }
+}
+//update brand information in db
+elseif(isset($_POST['update']))
+{    
+    $id = $_POST['id'];
+    $brand_id = $_POST['brand_id'];
+    $brand_name=$_POST['brand_name'];
+    $brand_description=$_POST['brand_description'];    
+    
+    // Check for empty fields
+    if(empty($brand_id) || empty($brand_name) || empty($brand_description)) {            
+        if(empty($brand_name)) {
+            echo "<font color='red'>Brand name field is empty.</font><br/>";
+        }
+        
+        if(empty($brand_description)) {
+            echo "<font color='red'>Brand description field is empty.</font><br/>";
+        }
+            
+    } else {    
+        //Update Query
+        $update_query = "UPDATE BRAND SET Brand_ID='$brand_id', Brand_Name='$brand_name', Brand_Desc='$brand_description' WHERE id=$id";
+        $update_result = mysqli_query($connection, $update_query);
+         $brands = mysqli_fetch_all($update_result, MYSQLI_ASSOC);
+        print_r($result);
+        
+        
```

Comment:
Some of this code seems to be what's in the updateDB() function in admin/model/brand.php file. Once you get it working it would be best to have the DB related logic in the function rather than on page. That way we can use the function wherever.